addons/mail/static/src/components/thread_view/thread_view.scss

```diff
@@ -13,6 +13,14 @@
     flex: 0 0 auto;
 }
 
+.o_ThreadView_loadingFailed {
+    display: flex;
+    flex: 1 1 auto;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+}
```

Comment:
@alexkuhn you tried on safari? Those children are not collapsed?

Reply:
Works fine on Safari, children not collapsed.